server/graphql/v2/input/CollectiveCreate.js

```diff
@@ -0,0 +1,14 @@
+import { GraphQLString, GraphQLNonNull, GraphQLList, GraphQLInputObjectType } from 'graphql';
+
+const CollectiveCreate = new GraphQLInputObjectType({
+  name: 'CollectiveCreate',
+  fields: () => ({
+    name: { type: new GraphQLNonNull(GraphQLString) },
+    slug: { type: new GraphQLNonNull(GraphQLString) },
+    description: { type: new GraphQLNonNull(GraphQLString) },
+    tags: { type: new GraphQLList(GraphQLString) },
+    githubHandle: { type: GraphQLString },
```

Comment:
@sbinlondon As collective params, there is also `githubHandle`.

Reply:
But this is only with open source projects, right?

Reply:
Yes